### PR TITLE
Server: Change asynchronous route to an ?async option.

### PR DESCRIPTION
**Pull request by gianm**

### CODE REVIEW

File: docs/server.md [hunk 1]
```diff
@@ -35,30 +35,17 @@ This API is synchronous and will block until your messages are either sent to Dr
...
-}
-```
-
-Note that "sent" will always be 0 when using the asynchronous API.
-
-If the server's outgoing message queue fills up, your requests will either block or fail. To control this behavior,
-set the `tranquility.blockOnFull` property to either true or false. If you set this to "false" and your server's
-outgoing queue is full, you will get an HTTP 503.
+If the server's outgoing message queue fills up, your asynchronous requests will either block or fail. To control this
+behavior, set the `tranquility.blockOnFull` property to either true or false. If you set this to "false" and your
```
Comment (nishantmonu51) [inline docs/server.md]: document the default value for tranquility.blockOnFull ?

File: docs/server.md [hunk 2]
```diff
@@ -35,30 +35,17 @@ This API is synchronous and will block until your messages are either sent to Dr
...
+server's outgoing queue is full, you will get an HTTP 503.
 
-With either the synchronous or asynchronous APIs, you can adjust the size of your Tranquility Server's message queues
-through the properties `tranquility.maxBatchSize` and `tranquility.maxPendingBatches`.
+With both the synchronous and asynchronous modes, you can adjust the size of your Server's message queues through
+the properties `tranquility.maxBatchSize` and `tranquility.maxPendingBatches`.
```
Comment (nishantmonu51) [inline docs/server.md]: would be nice to document the defaults if not already done.

Comment (gianm) [inline docs/server.md]: they're documented in configuration.md

Comment (gianm) [inline docs/server.md]: https://github.com/druid-io/tranquility/blob/master/docs/configuration.md

File: docs/server.md [hunk 3]
```diff
@@ -35,30 +35,17 @@ This API is synchronous and will block until your messages are either sent to Dr
 
 If there is an error sending the data, you will get an HTTP error code (4xx or 5xx).
 
-### Asynchronous HTTP API
+### Asynchronous option
 
-Tranquility Server also offers an asynchronous API. The form of the request should be the same as the synchronous API,
-but you should use the URLs `/v1/post-async` or `/v1/post-async/DATASOURCE`.
+Tranquility Server also offers an asynchronous option. Just add "?async=yes" to the URL. Note that "sent" in the
```
Comment (nishantmonu51) [inline docs/server.md]: should we change it to async=true instead of yes ?

Comment (nishantmonu51) [inline docs/server.md]: just to be consistent with other apis having boolean parameters.

Comment (gianm) [inline docs/server.md]: 'true', 'yes' and '1' are all supported, I can change the docs to 'true'

Comment (gianm) [inline docs/server.md]: ok, updated.

**Comment by dclim** [comment]
:+1:

**Pull request merged**